tests/form/robot/ValidationTextBox.html

```diff
@@ -495,13 +495,39 @@
 							doh.robot.typeKeys('123', 1000, 600);
 							doh.robot.sequence(d.getTestCallback(dojo.hitch(this, function(){
 								doh.is('123', this.textbox.focusNode.value, "focusNode.value");
-								doh.is(undefined, this.textbox.get('value'), "get('value')");
+								doh.is(123, this.textbox.get('value'), "get('value')");
 								doh.f(this.textbox.isValid(), "!isValid()");
```

Comment:
PS: I notice though that with the markup below, `isValid()` is true even before the blur:

```xml
<input type="text" data-dojo-type="dijit/form/NumberTextBox"
    data-dojo-props="constraints:{places:2}, editOptions:{places:'0,2'}">
```

So probably we should be consistent with that?